src/main/java/aeromon/command/AddCommand.java

```diff
@@ -0,0 +1,52 @@
+package aeromon.command;
+
+import aeromon.Storage;
+import aeromon.Ui;
+import aeromon.task.Deadline;
+import aeromon.task.Event;
+import aeromon.task.TaskArrayList;
+import aeromon.task.ToDo;
+
+import java.time.LocalDate;
+
+public class AddCommand extends Command {
+    private TaskType taskType;
+    private String[] info;
```

Comment:
Maybe can change the name of the field to tokens because info is not really descriptive enough for an array name